workspace/commands/merge.py

```diff
@@ -35,7 +35,7 @@ class Merge(AbstractCommand):
     :param bool quiet: Don't print merging if there are no commits to merge
     :param bool dry_run: Print out what will happen without making changes.
     :param str validation: A command to run after the merge and before a push to validate the change.
-    :param str with_ours: A lit of strings which will alter the commit merge strategy to ours if a commit message has
+    :param str skip_commits: A lit of strings which will alter the commit merge strategy to ours if a commit message has
```

Comment:
List of string or substrings from a commit message used to skip the commits during pint merge.

Reply:
sure